packages/dds/tree/src/feature-libraries/defaultTransaction.ts

```diff
@@ -25,9 +24,23 @@ class Transaction<TEditor extends ProgressiveEditBuilder<TChange>, TChange> {
         private readonly forest: IEditableForest,
         changeFamily: ChangeFamily<TEditor, TChange>,
     ) {
-        this.editor = changeFamily.buildEditor((delta) => {
-            this.forest.applyDelta(delta);
-        }, forest.anchors);
+        let currentRevision = 0;
+        const repairStore = new ForestRepairDataStore((revision: RevisionTag) => {
+            assert(
```

Comment:
Does this mean we cannot roll back transactions with more than one edit in them?

Reply:
No, `currentRevision` gets incremented every time a delta is applied to the forest.